meta/client.go

```diff
@@ -12,14 +12,18 @@ import (
 	"github.com/XiaoMi/pegasus-go-client/session"
 	"github.com/bluele/gcache"
 	"github.com/go-zookeeper/zk"
+	"github.com/pegasus-kv/meta-proxy/config"
+	"github.com/pegasus-kv/meta-proxy/metrics"
 	"github.com/sirupsen/logrus"
 )
 
-// TODO(jiashuo) store config file
-var zkAddrs = []string{""}
-var zkTimeOut = 1000000000 // unit ns, equal 1s
-var zkRoot = "/pegasus-cluster"
-var zkWatcherCount = 1024
+// declare perfcounters
+var tableWatcherEvictCounter metrics.Gauge
+
+var zkAddrs []string
+var zkRoot string
+var zkTimeOut int // unit ns
+var zkWatcherCount int
```

Comment:
Don't define configuration into variables unless you have to. Use `config.GlobalConfig.ZookeeperOpts.Address` directly.